feat: specified_id in dfx.json (#3565)
dfinity/sdk: CHANGELOG.md

# UNRELEASED

### feat: specified_id in dfx.json

> Aside from passing `--specified-id` in `dfx deploy` and `dfx canister create`, `specified_id` can be set in `dfx.json`.

Reviewer: ```suggestion
In addition to passing `--specified-id` in `dfx deploy` and `dfx canister create`, `specified_id` can be set in `dfx.json`.
```
